031-DevOpsWithGitHub/Student/challenge02.md

```diff
@@ -0,0 +1,44 @@
+# What The Hack: DevOps with GitHub
+
+## Challenge 2 - Repositories
+
+[< Previous](challenge01.md) - [Home](../readme.md) - [Next >](challenge03.md)
+
+### Introduction
+
+Historically, version control has been the first component that teams implement as they start on a project. It is one of the oldest and most well understood components of DevOps. Version control systems allow developers to colloborate and simulataneously contribute to the same codebase. They can also help teams track versions (so code can be rolled back if bad changes are made) and track bugs, work, and testing by the team. Please take a moment to review the [Git handbook](https://guides.github.com/introduction/git-handbook/) to understand the basics of version control, focusing on the distributed version control technology, Git.
+
+### Challenge
+
+Now that we have a basic understanding of version control and Git, lets get some code checked into source control. DevOps best practices can apply to any programming language, so for today we have provided you a simple .NET Core web application to use.
+
+1. Begin by cloning the GitHub repository you created in the [first challenge](challenge01.md) to your local computer ([hint](https://help.github.com/en/articles/cloning-a-repository)).
+
+2. Next, download the code for the application [here](https://github.com/OCPDemos/WhatTheHack/blob/027-DevOps-with-GitHub/027-DevOpsWithGitHub/Student/Code/Code.zip). Unzip the download and copy the contents into the root of the repository you just cloned.
```

Comment:
We discussed this on our review call.  Zip files should not be stored in the GH repo. 

We previously would have recommended using DownGit, but it's been glitchy, and occasionally blocked by Chrome as malware!

New recommendation is to ensure the lead coach zips up the contents of the /Student/Resources folder and post it to the Files tab of the hack event's team.   Here in the challenge text, it should say something like "Obtain the code from the Resource files provided by your coach"

Reply:
I removed the .zip file itself but forgot about that reference. Fixed!